runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java

```diff
@@ -123,8 +162,25 @@ protected void shutDown() throws Exception
 		worldMapPointManager.removeIf(MinigamePoint.class::isInstance);
 		worldMapPointManager.removeIf(FarmingPatchPoint.class::isInstance);
 		worldMapPointManager.removeIf(RareTreePoint.class::isInstance);
+		worldMapPointManager.removeIf(WayPointWorldMapPoint.class::isInstance);
+
 		agilityLevel = 0;
 		woodcuttingLevel = 0;
+
+		if (wayPoint != null)
+		{
+			client.clearHintArrow();
+		}
+	}
+
+	@Subscribe
+	public void onGameStateChanged(GameStateChanged event)
+	{
+		if (event.getGameState() == GameState.LOGGED_IN)
+		{
+			sendChatMessage(WAYPOINT_ACTIVE_MESSAGE);
```

Comment:
Yes. It seems the LOGGED_IN state always triggers after loading. The API says otherwise: "The user has successfully logged in." https://static.runelite.net/api/runelite-api/

Should I use a boolean and also use the LOGIN_SCREEN gamestate to allow the message to be displayed once?

Reply:
Yes. You might also need to check for `HOPPING` too.

Reply:
Should the message be displayed after a world hop?

Reply:
I don't care.